rfcs/0145-doc-strings.md

```diff
@@ -119,19 +117,157 @@ In the following we give a comprehensive overview to our decision that we've mad
 
 Adopting CommonMark as the content for all doc-comments brings the benefits of widely accepted and understood documentation format in tech projects, while maintaining profitability and consistency within the Nix ecosystem by aligning with existing [NixOS/RFC-72](https://github.com/NixOS/rfcs/blob/master/rfcs/0072-commonmark-docs.md).
 
-## `/**` starts a doc-comment
+The content indentation inherits the indentation logic from nixs' multiline strings `'' ''` allowing any arbitrary indentation the user might prefer.
```

Comment:
If the comment is formatted as Markdown, I think it is unnecessary to say we have the same "indentation" rules as Nix language. Because markdown itself strips indentation when displaying. 

For implementation concern, it's relatively difficult to maintain the same indentation rules (very complicated & constructed for the parser). I suggest to keep comments as-is when we lexed them, and just use any markdown render to display.